packages/server/src/utilities/plugins.js

```diff
@@ -0,0 +1,20 @@
+import env from "../environment"
+import { plugins as ProPlugins } from "@budibase/pro"
+
+export const enrichPluginURLs = plugins => {
+  if (!plugins || !plugins.length) {
+    return []
+  }
+  return plugins.map(plugin => {
+    const cloud = !env.SELF_HOSTED
+    // In self host we need to prefix the path, as "plugins" is not part of the
+    // bucket path. In cloud, "plugins" is already part of the bucket path.
+    let jsUrl = cloud ? "https://cdn.budi.live/" : "/plugins/"
```

Comment:
I believe there is a `PLUGIN_BUCKET_NAME` environment variable in backend-core - the `ObjectStoreBuckets` enumeration exported from `import { objectStore } from "@budibase/backend-core` automatically utilises this environment variable if it is specified. We probably need to account for this - in the cloud the bucket name doesn't matter as cloudfront makes this decision but in self host we should probably respect the env var.

Reply:
Thanks for pointing this out - didn't know it was configurable!